Source/Core/VideoCommon/VertexLoaderX64.cpp

```diff
@@ -115,7 +116,7 @@ int VertexLoaderX64::ReadVertex(OpArg data, u64 attribute, int format, int count
 
   X64Reg coords = XMM0;
 
-  int elem_size = 1 << (format / 2);
+  int elem_size = 1 << (u32(format) / 2);
```

Comment:
This could also be isolated into a `size_t GetElementSize(ComponentFormat format)` since it appears in a few places (with the same issue that it would be in the global namespace for the time being).